generators/common/templates/.lintstagedrc.js.ejs

```diff
@@ -16,15 +16,6 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -%>
-<%_
-  let handledExtensions = ['json', 'md', 'yml'];
-  if (!skipServer && prettierJava) {
-    handledExtensions.push('java');
-  }
-  if (!skipClient) {
-    handledExtensions.push('ts', 'css', 'scss');
-  }
-_%>
 module.exports = {
-  '{,src/**/}*.{<%= handledExtensions.join(',') %>}': ['prettier --write', 'git add']
+  '{,src/**/,webpack/}*.{<%= getPrettierExtensions() %>}': ['prettier --write', 'git add']
```

Comment:
`webpack` should not be included here if `skipClient` is true

Reply:
Done